management/src/main/java/org/kaazing/gateway/management/monitoring/entity/impl/AgronaMonitoringEntityFactory.java

```diff
@@ -39,17 +42,18 @@
 public class AgronaMonitoringEntityFactory implements MonitoringEntityFactory {
 
     private CountersManager countersManager;
+    private StringsManager stringsManager;
 
     // These are needed for the cleanup work that needs to be done in the close method.
     private File monitoringDirectory;
     private MappedByteBuffer mappedMonitorDirectory;
     private List<AtomicCounter> counters = new CopyOnWriteArrayList<AtomicCounter>();
+    private List<AtomicStringEntity> entities = new CopyOnWriteArrayList<AtomicStringEntity>();
```

Comment:
We could define this list name to something more specific (e.g. stringEntities), since counters could also be considered to be entities.